roles-in-tech/data-science.md

```diff
@@ -2,90 +2,90 @@
 
 Here are links to lessons that should be completed before this lesson:
 
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+- [Git version control](https://github.com/Techtonica/curriculum/blob/master/git-version-control/git-version-control.md)
+
+- [Python](https://github.com/Techtonica/curriculum/blob/master/python/python.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+*Data science is a sexy job. The salaries are high, the work is interesting, and there’s significant prestige that comes with the title.*
+
+* **[Bill James](https://en.wikipedia.org/wiki/Bill_James)** applied data analysis to baseball
+    * Who are the top performers? 
+    * How can you best predict future performance?
+
+* **Netflix** uses data analysis to recommend movies.
+
+<p align="center">
+	<img src="/roles-in-tech/data-example.jpg" alt="data">
+</p>
 
 ### Objectives
 
 **Participants will be able to:**
 
-- Things
-- More things
-- Even more things
-- Even more things
+- Analyze data
+- Learn from data
+- Use pandas and numpy
+- Predict Outcomes
 
 ### Specific Things To Teach
 
-- Things about the things
-- More things about the things
-	- This is a sub-thing about the things
-- Even more things about the things
-- Even more things about the things
+- Analyzing Data 
+- Cleaning Data using Pandas & Numpy
+	- Gaining insights
+- Building models on data
 
 ### Materials
 
-- [Example video (10 min)](https://example.com)
-- [Other example article(20 min read)](https://otherexample.com)
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under supplemental materials.) 
+- [Pandas Complete (1 hour video)](https://www.youtube.com/watch?v=vmEHCJofslg)
+- [Pandas (10 min read)](https://pandas.pydata.org/pandas-docs/stable/getting_started/10min.html)
+- [Numpy (20 min video)](https://www.youtube.com/watch?v=8Mpc9ukltVA)
 
 ### Lesson
 
-Here's text about introducing something and how it works.
-
-- [Be sure to link the slideshow, video,](google.com)
-- [or other materials you expect them to follow as part of the lesson.](google.com)
-
-Build on the first information. Have apprentices guess things, do an activity, etc.
-
-Make sure to mention these things:
+**[Data science](https://en.wikipedia.org/wiki/Data_science)** is a multi-disciplinary field that uses scientific methods, processes, algorithms and systems to extract knowledge and insights from data in various forms, both structured and unstructured, similar to data mining.
 
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
+**LifeCycle of Data Science**
+<p align="center">
+	<img src="/roles-in-tech/lifecycle-datascience.png" alt="data" height="500" width="500">
+</p>
```

Comment:
Google “markdown cheat sheet” to see how you can insert images using md notation!

Reply:
i am using it to align image to center.